sdlgui/progressbar.cpp

```diff
@@ -64,7 +64,6 @@ struct ProgressBar::AsyncTexture
       return;
 
     busy = true;
-    value = newvalue;
```

Comment:
It is shadowed in line 80/79... and this "newvalue" is not used because of call to `pbar->value()` on that lines...